src/test/java/ReceiptAnalysisTest.java

```diff
@@ -102,6 +102,16 @@
   private static final Optional<Long> TIMESTAMP_IN_1900S =
       Optional.of(Long.valueOf(INSTANT_IN_1900S.toEpochMilli()));
 
+  private static final String RAW_TEXT_WITH_PRICE = "the price is $12.77 in total";
```

Comment:
Have the string include the variable PRICE. This way it's easy to see that the asserted price is in the raw text.

Example: "the price is $" + PRICE + " in total".